docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -1879,6 +1879,54 @@ Current number of open HTTP connections for the node.
 `total_opened`::
 (integer)
 Total number of HTTP connections opened for the node.
+
+`clients`::
+(array of objects)
+Information on current and recently-closed HTTP client connections.
++
+.Properties of `clients`
+[%collapsible%open]
+=======
+`id`::
+(integer)
+Unique ID for the HTTP client.
+`agent`::
```

Comment:
Nit: I'd add a newline between each definition for readability. Won't affect the HTML output tho.